src/fsharp/FSharp.Core/list.fsi

```diff
@@ -338,6 +632,28 @@ namespace Microsoft.FSharp.Collections
         /// <param name="state">The initial state.</param>
         ///
         /// <returns>The state object after the folding function is applied to each element of the list.</returns>
+        /// 
+        /// <example id="foldback-example-1"> Shoping for fruits hungry, you tend to take more of each as the hunger grows
+        /// <code lang="fsharp">
+        /// type Fruit = Apple | Pear | Orange
+        /// type BagItem = { fruit: Fruit; quantity: int }
+        /// let takeMore fruit (previous: BagItem list) =
+        ///     let toTakeThisTime = 
+        ///         match previous with 
+        ///         | bagItem :: otherBagItems -> bagItem.quantity + 1 
```

Comment:
The sample isn't so great, because `otherBagItems` is not used.  Feels like it could drop out another way, not using fold

Reply:
This example is like this to show the difference between `fold` and `foldBack` behaviors. I just wanted to not put in the state the current quantity. I don't like this examples, but wanted to show something to Stefan and didn't came with a better example. Open to any other suggestion